apps/api/src/app/shared/framework/swagger/swagger.controller.ts

```diff
@@ -111,7 +112,8 @@ if (process.env.NOVU_ENTERPRISE === 'true') {
     { url: 'https://docs.novu.co/content-creation-design/translations' }
   );
 }
-export const setupSwagger = (app: INestApplication) => {
+export const setupSwagger = async (app: INestApplication) => {
+  await SwaggerModule.loadPluginMetadata(metadata);
```

Comment:
`metadata` is auto-generated by Swagger CLI during builds, it's necessary to provide back to the Swagger controller so it can create the Open API Spec successfully.